src/pages/components/content-switcher/style.mdx

```diff
@@ -68,10 +68,10 @@ plus the 16 px / 1rem on both sides of the text.
 
 ## Sizes
 
-| Size        | Height px / rem | 
-| ------------|---------------- |
-| Small (sm)  | 32 / 2          | 
-| Medium (md) | 40 / 2.5        | 
+| Size        | Height px / rem |
+| ----------- | --------------- |
+| Small (sm)  | 32 / 2          |
+| Medium (md) | 40 / 2.5        |
 | Large (lg)  | 48 / 3          |
 
 <div className="image--fixed">
```

Comment:
You need to move the structure image and caption here to be under the structure table.